shell.nix

```diff
@@ -59,7 +59,7 @@ let
         fi
       }
       PROMPT_COMMAND=prompt
-      export LC_ALL="C.UTF-8"
+      export LC_ALL=en_US.utf-8
```

Comment:
can you explain the difference to me?

Reply:
That change is wrong. I'll delete it.

On NixOS:
* this `export LC_ALL="C.UTF-8"` gives a warning, but everything still works
* `export LC_ALL=en_US.utf-8` this works just fine, no warnings

On other OSes in `nix-shell` this doesn 't work `LC_ALL=en_US.utf-8`

I haven't tried to figure out all the reasoning behind this, because waning on NixOS can just be ignored. As log as everything works I don't care much :smile:

Reply:
gotcha, thanks for the explanation! It would obviously be great to have no warnings, but indeed warnings are better than errors :) 

I'll make an issue so we can solve this later...

Reply:
Issue -> #2980

Reply:
Commit removed